### PR TITLE
Anonymous iframe OT post

### PR DESCRIPTION
**Pull request by mihajlija**
cc @agektmr 

Preview: https://deploy-preview-3611--developer-chrome-com.netlify.app/blog/anonymous-iframe-origin-trial/

Changes proposed in this pull request:

- Add Arthur as an author
- Add Anonymous iframe OT post

### CODE REVIEW

File: site/en/blog/anonymous-iframe-origin-trial/index.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+Register for the origin trial to enable your website to use Anonymous
+iframes:
+
+1.  [Request a token](/origintrials/#/view_trial/537054255563931649)
+    for your origin.
+1.  Use the token in one of the following ways:
+    -  In your HTML:
+
+      `<meta http-equiv="Origin-Trial" content="TOKEN_GOES_HERE">`
```
Comment (agektmr) [inline site/en/blog/anonymous-iframe-origin-trial/index.md:100]: Can't we remove this empty line + 2 spaces to align the sample code?

```suggestion
1.  Use the token in one of the following ways:
    -  In your HTML:  
      `<meta http-equiv="Origin-Trial" content="TOKEN_GOES_HERE">`
```

File: site/en/blog/anonymous-iframe-origin-trial/index.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+    -  In your Javascript:
+
+      ```js
+        const meta = document.createElement('meta');
+        meta.httpEquiv = 'Origin-Trial';
+        meta.content = 'TOKEN_GOES_HERE';
+        document.head.append(meta);
+      ```
```
Comment (agektmr) [inline site/en/blog/anonymous-iframe-origin-trial/index.md:109]: Same here.

```suggestion
    -  In your Javascript:  
      ```js
        const meta = document.createElement('meta');
        meta.httpEquiv = 'Origin-Trial';
        meta.content = 'TOKEN_GOES_HERE';
        document.head.append(meta);
      ```
```

File: site/en/blog/anonymous-iframe-origin-trial/index.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+    -  In the HTTP headers:
+
+     `Origin-Trial: TOKEN_GOES_HERE`
```
Comment (agektmr) [inline site/en/blog/anonymous-iframe-origin-trial/index.md:113]: Same here.

```suggestion
    -  In the HTTP headers:  
     `Origin-Trial: TOKEN_GOES_HERE`
```

File: site/en/blog/anonymous-iframe-origin-trial/index.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+1.  Add an anonymous iframe to your page:
+
+     `<iframe anonymous src="https://example.com">`
```
Comment (agektmr) [inline site/en/blog/anonymous-iframe-origin-trial/index.md:117]: Same here.

```suggestion
1.  Add an anonymous iframe to your page:  
     `<iframe anonymous src="https://example.com">`
```

**Review by agektmr — CHANGES_REQUESTED** [review]
Added some feedback.

File: site/en/blog/anonymous-iframe-origin-trial/index.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+## Register for an origin trial
+
+To ensure that Anonymous iframes are helping developers adopt cross origin
+isolation, we are making them available in Chrome from version 106 to 108 as an
+origin trial.
+
+Register for the origin trial to enable your website to use Anonymous
+iframes:
+
+1.  [Request a token](/origintrials/#/view_trial/537054255563931649)
```
[the remark this answers is not in the document]
Comment (mihajlija) [reply site/en/blog/anonymous-iframe-origin-trial/index.md:95]: I've seen the email thread, let's see if it gets resolved by tomorrow.

File: site/en/blog/anonymous-iframe-origin-trial/index.md (hunk 1 above)
Comment (mihajlija) [reply site/en/blog/anonymous-iframe-origin-trial/index.md]: Fixed, check the preview https://deploy-preview-3611--developer-chrome-com.netlify.app/blog/anonymous-iframe-origin-trial/

File: site/en/blog/anonymous-iframe-origin-trial/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (agektmr) [reply site/en/blog/anonymous-iframe-origin-trial/index.md:93]: Here's the updated link.

```suggestion
1.  [Request a token](/origintrials/#/view_trial/2518638091606949889)
```

**Review by agektmr — APPROVED** [review]
LGTM

File: site/en/blog/anonymous-iframe-origin-trial/index.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+
+## FAQ
+
+### Will this feature be adopted by other browsers?
+
+-   Mozilla Request for position:
+    [Pending](https://github.com/mozilla/standards-positions/issues/628)
+-   Webkit Request for position:
+    [No signal](https://lists.webkit.org/pipermail/webkit-dev/2022-April/032205.html)
```
Comment (agektmr) [inline site/en/blog/anonymous-iframe-origin-trial/index.md:135]:
```suggestion
    [No signal](https://github.com/WebKit/standards-positions/issues/45)
```

**Pull request merged**